README.md

```diff
@@ -244,6 +244,40 @@ type Output struct {
 }
 ```
 
+### Struct field name folding
+
+With `goverter:fold` tag you can instruct goverter to perform case-insensitive mapping between
+source and target fields. This is useful when converting between two structs auto-generated by
+two different tools, each following different name casing rules for struct fields: goverter
+will match the fields using strings.EqualFold method if fold tag is present.
+
+Use this tag only when it is extremely unlikely for the source or the target to have two fields
+that differ only in casing. E.g.: converting go-jet generated model to protoc generated struct.
+If fold is present and goverter detects ambiquous match, it will either prefer an exact match if
+such is found or report an error. Use either goverter:map to make an explicit mapping choice
+or goverter:ignore to ignore irrelevant ambiguous fields.
```

Comment:
I think we shouldn't recommend goverter:ignore to fix the ambiguous match, because explicitly mapping via goverter:map should always be better. Please remove it from the code example and the text here (the implementation should stay the same)

Reply:
ack, removed the recommendation from here. I still kept the implementation and its test case to avoid breaking change if somebody ends up using it.